support-workers/src/main/scala/com/gu/emailservices/DigitalPackEmailFields.scala

```diff
@@ -180,7 +184,7 @@ class DigitalPackEmailFields(
     ))
 
   private def corpRedemption =
-    wrap("digipack-corp", directOrCorpFields("Group subscription"))
+    wrap("digipack-corporate-redemption", directOrCorpFields("Group subscription"))
```

Comment:
I decided to change this at the same time to make it more consistent
https://github.com/guardian/membership-workflow/pull/270/files#diff-9145604e78c7f805b7bdb0c49e8785baR54